src/sentry/static/sentry/app/views/organizationIntegrations/providerRow.tsx

```diff
@@ -111,12 +112,16 @@ export default class ProviderRow extends React.Component<Props> {
   }
 
   render() {
+    const isBeta = this.props.provider.key === 'pagerduty' ? true : false;
```

Comment:
You can simplify this to

```suggestion
    const isBeta = this.props.provider.key === 'pagerduty';
```